boards/xtensa/esp32_net/doc/index.rst

```diff
@@ -0,0 +1,153 @@
+.. _esp32_net:
+
+ESP32-NET
+#########
+
+Overview
+********
+
+ESP32_NET is a board configuration to allow zephyr application building
+targeted to ESP32 APP_CPU only, please refer ESP32 board to a more complete
+list of features.
+
+System requirements
+*******************
+
+Prerequisites
+=============
+
+The ESP32 toolchain :file:`xtensa-esp32-elf` is required to build this port.
```

Comment:
The content in this file needs to be updated. Check latest `boards/xtensa/esp32/doc/index.rst`.  
Currently, ESP32 toolchain is provided in Zephyr's SDK.